posthog/management/commands/cleanup_expired_sharing_configs.py

```diff
@@ -0,0 +1,40 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from posthog.models.sharing_configuration import SharingConfiguration
+import structlog
+
+logger = structlog.get_logger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
Why do this as a command and not as a scheduled job or something else? I feel that this won't be run often because people will likely forget.